scripts/create-new-version

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+echo "Welcome to the guided process of updating the Guides search index"
+echo "We will now be running a mix of automated (🤖) and manual (👩‍💻) steps, please read the prompts carefully."
+
+STARTING_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+TEMP_BRANCH="create-new-guides-version-$(( $RANDOM % 1000 ))"
+
+echo
+echo "🤖 Staging local changes"
+git add .
+git stash push -m $TEMP_BRANCH
+echo "  DONE"
+
+echo
+echo "🤖 Updating master branch from remote"
+read -p "Remote name (origin): " REMOTE
+REMOTE_NAME=${REMOTE:-origin}
+git fetch $REMOTE_NAME master
+git checkout FETCH_HEAD -b $TEMP_BRANCH
+echo "  DONE"
+
+echo
+echo "👩‍💻 Merge any pending PRs for the current release https://github.com/ember-learn/guides-source/pulls"
+read -n 1 -s -r -p "Press any key to continue"
+
+CURRENT_VERSION=$(grep "currentVersion" guides/versions.yml | sed "s/currentVersion.*\(v.*\)\"/\1/")
+MAJOR_VERSION=$(echo $CURRENT_VERSION | cut -d "." -f 1 | cut -c 2-)
+MINOR_VERSION=$(echo $CURRENT_VERSION | cut -d "." -f 2)
+PATCH_VERSION=$(echo $CURRENT_VERSION | cut -d "." -f 3)
+NEXT_VERSION=$(($MAJOR_VERSION.$(($MINOR_VERSION+1)).$PATCH_VERSION))
```

Comment:
It looks like this line of code can only handle a minor version release.

What does a contributor do if there is a major version release? Do they manually create a subfolder in `guides` as they might do currently?

Reply:
I decided to focus on minor releases since I didn't want to handle conditionals in the bash script, and we have only had two major releases so far and they might require more coordination already as it is.
The idea would be to add the functionality of major releases in a following effort. You raise a good point, we should say the script currently does patch releases.

Reply:
I think adding the functionality for major release at a later point is a good idea.

In a follow-up PR, we may want to update `MAINTAINERS.md` (revert some, but not necessarily all, of steps 2-8) to describe what to do when it's time for a major release.